service-commands/scripts/userClockValues.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Find the creator nodes associated with a user and their respective clock values.
+ *
+ * Script usage: node userClockValues.js <size> <outFile>
+ */
+const axios = require('axios')
+
+const discoveryProvider = 'https://discoveryprovider.audius.co'
```

Comment:
can we retrieve discovery provider through libs instead of hardcoding it here?

Reply:
Yeah, that makes sense, I'll update.

Reply:
I'm not sure if we can get it via libs, because I couldn't find a reference to it anywhere in the code.

I've changed it to an env variable similar to how Vicky has handled it in https://github.com/AudiusProject/audius-protocol/pull/1021 https://github.com/AudiusProject/audius-protocol/blob/74223efaa089eb548da50b7ce3bcb3504f195d6c/creator-node/scripts/updateContentBlacklist.js#L15